client/landing/stepper/declarative-flow/internals/steps-repository/intro/index.tsx

```diff
@@ -16,8 +18,24 @@ import './styles.scss';
 
 const useIntroContent = ( flowName: string | null ): IntroContent => {
 	const { __ } = useI18n();
-
+	const urlQueryParams = useQuery();
 	return useMemo( () => {
+		if ( isCopySiteFlow( flowName ) ) {
+			return {
+				title: __( 'Copy Site' ),
+				text: createInterpolateElement(
+					__(
+						'You’re 5 minutes away from<br />creating a new copy site from <SourceUrl/>.<br />Ready?'
+					),
+					{
+						br: <br />,
+						SourceUrl: <span>{ urlQueryParams.get( 'sourceUrl' ) }</span>,
+					}
+				),
+				buttonText: __( 'Start copying' ),
+			};
+		}
```

Comment:
I created an issue for refining this: https://github.com/Automattic/dotcom-forge/issues/1479